data.md

```diff
@@ -158,6 +158,11 @@ Data Structures
 WASM is a stack-machine, so here we provide the stack to operate over.
 Operator `_++_` implements an append operator for sort `Stack`.
 
+Integers in K are unbounded. As an invariant, however, for any integer `< iNN >
+I:Int` on the stack, `I` is between 0 and `#pow(NN) - 1`. That way, unsigned
+instructions can make use of `I` directly, whereas signed instructions may need
+`#signed(iNN, I)`.
```

Comment:
`data.md` is more for specifying underlying data-structures, and it's not an invariant of the `Stack` sort that this is true, only of the `<stack>` cell. I would move it to `wasm.md` under a new section called `### Assumptions and Invariants` under the `Configuration` subsection.

Also the rest of the repo does 1 sentence/line (and no line breaks otherwise). Let's keep following the convention.

Reply:
Fixed.